tests/conanfile.txt

```diff
@@ -0,0 +1,9 @@
+[requires]
+cpprestsdk/2.10.14@bincrafters/stable
+nlohmann_json/3.7.3
+
+[options]
+libzip:with_openssl=False
+
+[generators]
+cmake
```

Comment:
Is boost not required here?

Reply:
That's a dependency of CPPRestSDK

Reply:
It may not be best practice to rely on it being a dependency. I doubt it will change anything if you add it, but it would make things easier if we ever decide to change libs.